src/components/PeopleList.tsx

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import cn from 'classnames';
+import { Person } from '../types/Person';
+
+type Props = {
+  people: Person[],
+  onSelected: (person: Person) => void,
+};
+
+export const PeopleList: React.FC<Props> = ({ people, onSelected }) => {
+  return (
+    <div className="dropdown-menu" role="menu">
+      <div className="dropdown-content">
+        {people.length !== 0
+          ? people.map(person => (
+            <a
+              href="/#"
+              className="dropdown-item"
+              onClick={event => {
+                event.preventDefault();
+                onSelected(person);
+              }}
```

Comment:
better to create separate function
```suggestion
              onClick={onSelect}
```